### PR TITLE
Updated collaboration model readme

### PR DESCRIPTION
**Pull request by DennisCES**

### CODE REVIEW

**Comment by philecs** [comment]
We need to update all reference of `collaborationelement` to `collaborationElement`. And the file itself must be renamed to `collaborationElement`

**Comment by philecs** [comment]
> schema .\calendarEvent.json is invalid
> error: can't resolve reference #/definitions/attachment from id https://github.com/openintegrationhub/Data-and-Domain-Models/blob/master/src/main/schema/collaboration/collaborationElement.json#

See line 118 in collaborationElement.json

**Comment by philecs** [comment]
We also need a json schema for the `taskToTaskRelation` object. Currently missing

**Comment by philecs** [comment]
Within UML diagram: `taskToTaskRelation` should have two additional properties namely sourceUid and targetUid

File: MasterDataModels/Collaborations/CollaborationModelDescription.md [hunk 1]
```diff
@@ -16,19 +16,19 @@
...
 |end|Date|-|enddate of the event|31.12.2018|-|
 
-## Contacts
+## Contact
 |Attribute|Type|Properties|Description|Example|Possible Enumeration Options|
 |---|---|---|---|---|---|
 |name|String|-|Name of the contact|"John Doe"|-|
 |eMail|String|-|E-mail of the contact|"john.doe@email.com"|-|
-|[calendars](#calendar)|Array (of calendar objects)|-|Calendar information of the person|A calendar object array|-|
+|[calendar](#calendar)|Array (of calendar objects)|-|Calendar information of the person|A calendar object array|-|
```
[the remark this answers is not in the document]
Comment (DennisCES) [reply MasterDataModels/Collaborations/CollaborationModelDescription.md]: Done

File: MasterDataModels/Collaborations/CollaborationModelDescription.md [hunk 2]
```diff
@@ -40,8 +40,8 @@
 ## Email
 |Attribute|Type|Properties|Description|Example|Possible Enumeration Options|
 |---|---|---|---|---|---|
-|[collaborationElement](#collaborationelement)|Collaboration object|-|Collaboration properties values|A collaborationElement object array|-|
-|[threads](#thread)|Array (of thread objects)|-|thread information of the e-mail|A thread object array|-|
+|[collaborationElement](#collaborationelement)|Array (of collaboration objects)|-|Collaboration properties values|A collaborationElement object array|-|
+|[thread](#thread)|Array (of thread objects)|-|thread information of the e-mail|A thread object array|-|
```
[the remark this answers is not in the document]
Comment (DennisCES) [reply MasterDataModels/Collaborations/CollaborationModelDescription.md]: Done

File: src/main/schema/collaboration/calendarEvent.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
       "description":"CollaborationElement Data",
       "allOf": [
         {
-          "$ref":"collaborationelement.json"
+          "$ref":"collaborationElement.json"
         }
```
[the remark this answers is not in the document]
Comment (DennisCES) [reply src/main/schema/collaboration/calendarEvent.json]: Done

File: MasterDataModels/Collaborations/CollaborationModelDescription.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (philecs) [reply MasterDataModels/Collaborations/CollaborationModelDescription.md]: not yet 🗡

**Pull request merged**